### PR TITLE
ccc 2022

### PR DESCRIPTION
**Pull request by sequencer**
### Contributor Checklist

- [ ] Did you add Scaladoc to every public function/method?
- [ ] Did you add at least one test demonstrating the PR?
- [ ] Did you delete any extraneous printlns/debugging code?
- [ ] Did you specify the type of improvement?
- [ ] Did you add appropriate documentation in `docs/src`?
- [ ] Did you state the API impact?
- [ ] Did you specify the code generation impact?
- [ ] Did you request a desired merge strategy?
- [ ] Did you add text to be included in the Release Notes for this change?

### Reviewer Checklist (only modified by reviewer)
- [ ] Did you add the appropriate labels?
- [ ] Did you mark the proper milestone (Bug fix: `3.4.x`, [small] API extension: `3.5.x`, API modification or big change: `3.6.0`)?
- [ ] Did you review?
- [ ] Did you check whether all relevant Contributor checkboxes have been checked?
- [ ] Did you do one of the following when ready to merge:
  - [ ] Squash: You/ the contributor `Enable auto-merge (squash)`, clean up the commit message, and label with `Please Merge`.
  - [ ] Merge: Ensure that contributor has cleaned up their commit history, then merge with `Create a merge commit`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+1. Constellation, a Open-source Chisel NoC Generator for SoCs - Jerry Zhao@UCB BAR
+2. The formal verification capabilities of chiseltest - Kevin Laeufer@UCB BAR
+3. Chisel Breakdown 03 - Jack Koenig@SiFive
+4. The Next Generation FIRRTL Compiler is Here! - Prithayan Barua&Hideto Ueno@SiFive
+5. Implementing RISC-V Scalar Cryptography/Bitmanip extensions in Chisel - Hongren Zheng@Tsinghua University+PLCT
+6. SCIRT: Bridging the Type System Gap for Circuit Generators - Ruikang Wang@Tsinghua University+PLCT
+7. ChiselDB: Mapping Hardware Data Structures to Database Tables for Efficient Data Analysis Jiawei Lin@ICT
+8. From Chisel to Chips in Fully Open-Source - Martin Schoeberl@DTU
+
+See you this week :)
```
Comment (jackkoenig) [inline README.md:23]:
```suggestion
See you this week :)

```
Need a newline here or this becomes a title

File: README.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
 ![Chisel 3](https://raw.githubusercontent.com/chipsalliance/chisel3/master/docs/src/images/chisel_logo.svg?sanitize=true)
 
+---
+### CCC 2022
+The annual CCC (Chisel Community Conference) will be held online, You can add [ics file](https://calendar.google.com/calendar/embed?src=c_pluc02j9c4ambkiljlud3620fk%40group.calendar.google.com&ctz=Atlantic%2FAzores) to the calendar for conference agenda.
+
+Saturday, August 27 ⋅ 09:00 - 16:00 GMT +8(Shanghai/China);
+Friday, August 26 ⋅ 18:00 - 25:00 GMT -7(Los Angeles/US);
+Saturday, August 27 ⋅ 02:00 - 9:00 GMT +1(London/UK);
```
Comment (edwardcwang) [inline README.md:9]:
```suggestion
Saturday, August 27 ⋅ 09:00 - 16:00 UTC+8 (Shanghai/China);
Friday, August 26 ⋅ 21:00 - 04:00 UTC-4 (Eastern Daylight Time);
Friday, August 26 ⋅ 18:00 - 25:00 UTC-7 (Los Angeles/US);
Saturday, August 27 ⋅ 02:00 - 9:00 UTC+1 (London/UK);
```

File: README.md [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+---
+### CCC 2022
+CCC(Chisel Community Conference) is an annual gathering of Chisel community enthusiasts and technical exchange workshop. With the support of the Chisel development community and RISC-V International, this conference will bring together designers and developers with hands-on experience in Chisel from home and abroad to share cutting-edge results and experiences from both the open source community as well as industry.
+
+CCC2022 will be held online, You can add [ics file](https://calendar.google.com/calendar/embed?src=c_pluc02j9c4ambkiljlud3620fk%40group.calendar.google.com&ctz=Atlantic%2FAzores) to the calendar for conference agenda.
+
+Saturday, August 27 ⋅ 09:00 - 16:00 UTC+8 (Shanghai/China);
+Friday, August 26 ⋅ 21:00 - 04:00 UTC-4 (Eastern Daylight Time);
+Friday, August 26 ⋅ 18:00 - 25:00 UTC-7 (Los Angeles/US);
+Saturday, August 27 ⋅ 02:00 - 9:00 UTC+1 (London/UK);
```
Comment (jackkoenig) [inline README.md:12]:
```suggestion
Conference Time (in various time zones)
* Saturday, August 27 ⋅ 09:00 - 16:00 UTC+8 (Shanghai/China)
* Friday, August 26 ⋅ 21:00 - 04:00 UTC-4 (Eastern Daylight Time)
* Friday, August 26 ⋅ 18:00 - 25:00 UTC-7 (Los Angeles/US)
* Saturday, August 27 ⋅ 02:00 - 9:00 UTC+1 (London/UK)
```

Thoughts on tweaking the formatting of this part a little bit?

Comment (sequencer) [reply README.md]: Sure! Thanks for doing that!

**Review by jackkoenig — APPROVED** [review]

**Pull request merged**